align/compiler/find_constraint.py

```diff
@@ -287,9 +287,8 @@ def FindConst(subckt):
                 isinstance(const, constraint.GroundPorts) or \
                 isinstance(const, constraint.ClockPorts):
             stop_points.update(const.ports)
-        elif isinstance(const, constraint.IsDigital) or \
-                isinstance(const, constraint.AutoConstraint):
-            auto_constraint = const.isTrue
+        elif isinstance(const, constraint.ConfigureCompiler):
+            auto_constraint = not getattr(const, 'is_digital', False) and getattr(const, 'auto_constraint', False)
```

Comment:
These getattr's can be removed now. The default value can be directly pulled. Example:
```python
auto_constraint = const.is_digital and const.auto_constraint
```

Reply:
Done.